File: DiscordWebhookMessageSender.cs
using System;
using System.Net;
using System.IO;

namespace SendMessageToDiscord
{
    class Program
    {
        static void Main(string[] args)
        {
            // Replace webhook url with your own
            string webhookurl = "WEBHOOK_URL_GOES_HERE";

            // Replace with your own message
            string message = "Hello, this is a message sent from C# code!";

            // Create a web request to the Discord webhook url
            HttpWebRequest request = WebRequest.Create(webhookurl) as HttpWebRequest;

            // Set the request method, content type and the content length
            request.Method = "POST";
            request.ContentType = "application/json";
            request.ContentLength = message.Length;

            // Write the request body
            using (StreamWriter writer = new StreamWriter(request.GetRequestStream()))
            {
                writer.Write(message);
            }

            // Get the response and print it
            HttpWebResponse response = (HttpWebResponse)request.GetResponse();
            string result = null;
            using (StreamReader reader = new StreamReader(response.GetResponseStream()))
            {
                result = reader.ReadToEnd();
            }

            Console.WriteLine(result);
        }
    }
}

File: DiscordWebhookMessageSender.js
using System;
using System.Net;
using System.IO;

<script type="text/javascript">
    // Replace the following variables with your own values
    const webhook = 'https://discordapp.com/api/webhooks/XXXXXXXXXX/XXXXXXXXXX'; // Replace with your webhook URL
    const message = 'Hello! This is a message sent by a JavaScript code!'; // Message to be sent
    
    // Don't change anything below this line
    const data = {
      "content": message
    };
    fetch(webhook, {
      method: 'POST',
      body: JSON.stringify(data),
      headers: {
        'Content-Type': 'application/json'
      }
    })
    .then(res => console.log('Message sent!'))
    .catch(err => console.error(err));
</script>